tox.ini

```diff
@@ -1,7 +1,16 @@
 [tox]
-envlist = py26, py27, py32, py33, py34, pypy, pypy3
+envlist = py26, py27, py27-docs, py32, py33, py34, pypy, pypy3
```

Comment:
Change this to `docs`, to match the env you declared lower. I prefer the `docs` name, so use that.